src/core_plugins/kibana/public/discover/embeddable/search_embeddable.js

```diff
@@ -77,11 +85,23 @@ export class SearchEmbeddable extends Embeddable {
   initializeSearchScope() {
     this.searchScope = this.$rootScope.$new();
 
-    this.pushContainerStateParamsToScope();
-
     this.searchScope.description = this.savedSearch.description;
     this.searchScope.searchSource = this.savedSearch.searchSource;
 
+    const timeRangeSearchSource = this.searchScope.searchSource.new();
+    timeRangeSearchSource.filter(() => {
+      return getTime(this.searchScope.searchSource.index(), this.timeRange);
+    });
+
+    const filtersSearchSource = this.searchScope.searchSource.new();
```

Comment:
We should add a bit more of inline docs in the end to this :-) Explaining the different search sources and highlight, that we will set the query and filters later via the pushContainerStateParams.